lib/helper/desktopHelper.py

```diff
@@ -38,8 +38,10 @@ def launch_browser(browser_type, **kwargs):
 
     if env.PROFILER_FLAG_CHROMETRACING in enabled_profiler_list:
```

Comment:
what about:
```
if browser_type == env.DEFAULT_BROWSER_TYPE_CHROME:
    if env.PROFILER_FLAG_CHROMETRACING in enabled_profiler_list:
        profile_path = env.chrome_profile_path
        browser_obj = chrome_class(env.DEFAULT_BROWSER_HEIGHT, env.DEFAULT_BROWSER_WIDTH,
                                   tracing_path=env.chrome_tracing_file_fp,
                                   profile_path=profile_path)
    else:
        profile_path = env.chrome_profile_path
        browser_obj = chrome_class(env.DEFAULT_BROWSER_HEIGHT, env.DEFAULT_BROWSER_WIDTH, profile_path=profile_path)

elif browser_type == env.DEFAULT_BROWSER_TYPE_FIREFOX:
    if env.PROFILER_FLAG_FXTRACELOGGER in enabled_profiler_list:
        profile_path = env.firefox_profile_path
        browser_obj = firefox_class(env.DEFAULT_BROWSER_HEIGHT, env.DEFAULT_BROWSER_WIDTH, tracelogger=True,
                                    profile_path=profile_path)
    else:
        profile_path = env.firefox_profile_path
        browser_obj = firefox_class(env.DEFAULT_BROWSER_HEIGHT, env.DEFAULT_BROWSER_WIDTH,
                                    profile_path=profile_path)
```

This might reduce some duplicated code.